### PR TITLE
Add ARM64 Support

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -37,6 +37,45 @@ steps:
...
+  arch: arm64
+
+steps:
+- name: build
+  image: docker:dind
+  volumes:
+  - name: dockersock
+    path: /var/run
+  commands:
+  - sleep 20
```
Comment (galal-hussein) [inline .drone.yml:54]: any reason the sleep here is needed?

Comment (briandowns) [reply .drone.yml:54]: Copy pasta from amd64. It might not be needed but kept it to be safe.

**Review by galal-hussein — APPROVED** [review]

**Review by VestigeJ — APPROVED** [review]
LGTM

File: Dockerfile.s390x [hunk 2]
```diff
@@ -1,7 +1,7 @@
-ARG GOLANG_VERSION=1.19.0-alpine3.15
+ARG GO_VERSION=1.19.0-alpine3.15
```
Comment (brandond) [inline Dockerfile.s390x:1]: I'm really confused about how this is working, in the other dockerfiles GO_VERSION is just the bare go version, here we have the -alpine suffix on it, and then we add -alpine again on line 4?

Comment (brandond) [reply Dockerfile.s390x:1]: The GOLANG_VERSION arg is already used in the library/golang image; perhaps there was some interaction with that?

https://github.com/docker-library/golang/blob/master/1.20/alpine3.17/Dockerfile#L13

**Review by brandond — CHANGES_REQUESTED** [review]
Got a question on s390x inconsistency

File: Dockerfile.s390x (hunk 2 above)
Comment (briandowns) [reply Dockerfile.s390x]: env var refs updated.

**Review by brandond — APPROVED** [review]

**Pull request merged**